r3bbase/R3BFileSource.cxx

```diff
@@ -427,26 +458,38 @@ void R3BFileSource::SetInTree(TTree* tempTree)
     Init();
 }
 
-// ----  Method getrunid -----------------------------------------------------
-Int_t R3BFileSource::getrunid(uint64_t st)
+// ----  Method GetRunid -----------------------------------------------------
+Int_t R3BFileSource::GetRunid(uint64_t st)
 {
-    for (Int_t i = 0; i < array_size - 1; i++)
-        if (st >= ftimestamp[i] && st < ftimestamp[i + 1])
+    UInt_t fArray = fTimestamp.size();
```

Comment:
Better use 'auto' type for fArray. Name suggestion: fArray -> fArraySize or fNumTimestamps or fNumEntries.

Reply:
Yes, you are right! Thanks.